### PR TITLE
Governance: Implement Absolute max voter weight source

### PR DESCRIPTION
**Pull request by SebastianBor**
#### Summary 

Implement `Absolute` option for `community_mint_max_voter_weight_source` to support static values not related to the governance token supply

### CODE REVIEW

File: governance/program/src/state/proposal.rs [hunk 1]
```diff
@@ -1834,6 +1832,57 @@ mod test {
         assert_eq!(proposal.max_vote_weight, Some(100));
     }
 
+    #[test]
+    fn test_try_tip_vote_with_reduced_absolute_community_mint_max_vote_weight() {
+        // Arrange
+        let mut proposal = create_test_proposal();
+
+        proposal.options[0].vote_weight = 60;
```
Comment (joncinque) [inline governance/program/src/state/proposal.rs:1840]: As another test, can you try vote with more than the absolute voter weight?  For example, just like this one, but instead have `vote_weight = 120`.  Just to make sure that overflows are correctly handled

**Review by joncinque — COMMENTED** [review]
The logic looks good, but I'd like to see another test for the "overflow" case of a low absolute max and a lot of tokens used.

File: governance/program/src/state/proposal.rs (hunk 1 above)
Comment (SebastianBor) [reply governance/program/src/state/proposal.rs:1840]: Done

**Review by joncinque — APPROVED** [review]
Perfect, thanks!
